tests/cover/test_reflection.py

```diff
@@ -556,9 +558,22 @@ def test_does_not_put_eval_directory_on_path():
     assert eval_directory() not in sys.path
 
 
```

Comment:
This function change was not related to the bug fix - it was just a style change on my part.  I felt that `varargs` was better represented by a function defined in the test rather than a function mixed in with all the tests